doc/CONTRIBUTING.md

```diff
@@ -72,14 +75,14 @@ If you already have a Python development environment on your machine and a prefe
 
     **Note:** If you're running multiple versions of Python on your machine, you can make sure your data act environment is running the correct Python version by pointing to a specific binary
 
-        mkvirtualenv --python=[path to installed Python 2.7 executable] data-act
+        mkvirtualenv --python=[path to installed Python 3.x executable] data-act
 
 6. You should see some output that looks similar to the example below. Essentially, this command creates and activates a new virtualenv named `data-act` with its own set of Python libraries.  Anything you pip install from this point forward will be installed into the *data-act* environment rather than your machine's global Python environment. Your command line prompt indicates which (if any) virtualenv is active.
 
         rebeccasweger@GSA-xs-MacBook-Pro-4 ~
-        $ mkvirtualenv --python=/usr/local/bin/python2.7 data-act
-        Running virtualenv with interpreter /usr/local/bin/python2.7
-        New python executable in data-act-test/bin/python2.7
+        $ mkvirtualenv --python=/usr/local/bin/python3.4 data-act
+        Running virtualenv with interpreter /usr/local/bin/python3.4
+        New python executable in data-act-test/bin/python3.4
```

Comment:
Good call on removing the extraneous terminal info. I intended the `mkvirtualenv` command to show a real implementation (there's a note above that the `--python` parameter should point to whatever Python executable you want to use).

I personally find it helpful to see how a command would work in practice but don't want to confuse anyone. What do you think?